benchmark/buffers/buffer-copy.js

```diff
@@ -4,16 +4,18 @@ const common = require('../common.js');
 const bench = common.createBenchmark(main, {
   bytes: [0, 8, 128, 32 * 1024],
   partial: ['true', 'false'],
+  oob: ['true', 'false'],
```

Comment:
It is uncommon to to declare an oob source end. I do not think we should keep a benchmark configuration for this edge case.

It is good to know that it's improved but keeping it in code means this benchmark is going to take twice as long without providing a real benefit for the average case in the future. Please remove the benchmark changes therefore.